vizro-core/tests/integration/test_examples.py

```diff
@@ -48,6 +48,7 @@ def dashboard(request, monkeypatch):
 @pytest.mark.filterwarnings("ignore:HTTPResponse.getheader()")
```

Comment:
```suggestion
@pytest.mark.filterwarnings("ignore:HTTPResponse.getheader():DeprecationWarning")
```

Reply:
Tested.
Working only like this
```suggestion
@pytest.mark.filterwarnings("ignore:HTTPResponse.getheader():DeprecationWarning")
```